server/models/users.js

```diff
@@ -347,15 +357,20 @@ module.exports = class User extends Model {
       }
     }
     console.info(redirect)
+    WIKI.logger.info(`users.afterLoginChecks: Group lookup and redirect completed`)
+    WIKI.logger.info(redirect)
```

Comment:
is this different from the console.info(redirect) two lines above

Reply:
console.info doesn't actually output anything unless you're running locally